css/styles.css

```diff
@@ -0,0 +1,188 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'San Fransisco', Arial, Helvetica, sans-serif;
+}
+
+.nav-bar {
+  background-image: linear-gradient(#999, #777 50%, #666 50%, #888);
+  border-radius: 4px;
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px;
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  height: 36px;
+  top: 90px;
+  margin: 0 auto;
+  position: relative;
+  display: block;
+  width: 980px;
+  z-index: 0;
+}
+
+.list {
+  height: 100%;
+  display: flex;
+  list-style: none;
+  align-items: center;
+  padding: 0;
+}
+
+.image {
+  height: 23px;
+  width: 23px;
+}
+
+.form {
+  height: 36px;
+  width: 120px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 3px;
+}
+
+.input {
+  width: 80%;
+  background-image: linear-gradient(#999, #777 50%, #666 50%, #888);
+  border-radius: 18px;
+  padding: 2px 18px 1px 5px;
+  height: 51%;
+  box-shadow: rgba(0, 0, 0, 0.6) 0 2px 1px inset, rgba(255, 255, 255, 0.2) 0 1px 0;
+  border: none;
+}
+
+.list-link {
+  text-decoration: none;
+  color: #fff;
+}
+
+.list-items {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-right: 1px solid #888;
+  background-image: linear-gradient(#878787, #666 50%, #444 50%, #777);
+}
+
+.list-items:hover {
+  background: linear-gradient(to bottom, rgba(71, 71, 71, 1) 0%, rgba(92, 92, 92, 1) 50%, rgba(66, 66, 66, 1) 51%, rgba(66, 66, 66, 1) 100%);
+}
+
+.header {
+  background: url(../Images/alpin.jpg) 50% 98.9% no-repeat;
+  display: block;
+  width: 100%;
+  height: 538px;
+}
+
+body {
+  margin-top: -26px;
+  width: 100%;
+  margin-left: -1px;
+}
+
+.bold {
+  display: flex;
+  justify-content: center;
+}
+
+.h-link {
+  display: block;
+  margin-top: 120px;
+  color: #fff;
+  text-decoration: none;
+  font-weight: 200;
+  text-align: center;
+  font-size: 63px;
+  line-height: 36px;
+  font-style: normal;
+  margin-bottom: 6px;
+}
+
+.medium {
+  display: flex;
+  justify-content: center;
+}
+
+.h-link1 {
+  display: block;
+  color: #fff;
+  text-decoration: none;
+  width: 800px;
+  text-align: center;
+  font-size: 32px;
+  font-weight: 200;
+  font-style: normal;
+  margin-top: 4px;
+}
+
+.play-button {
+  width: 25px;
+  height: 25px;
+  padding-left: 7px;
+  margin-top: 21px;
+}
+
+.h-link2 {
+  color: #fff;
+  text-decoration: none;
+  text-align: center;
+  font-weight: 200;
+  display: inline-block;
+  margin-top: 19px;
+  font-size: 25px;
+}
+
+.small {
+  display: flex;
+  justify-content: center;
+}
+
+.h-link2:hover {
+  text-decoration: underline;
+}
+
+.aside {
+  z-index: 1;
+  bottom: 90px;
+  width: 100%;
+  height: 200px;
+  display: flex;
+}
+
+.promo {
+  width: 25%;
+  margin-top: 2px;
+}
+
```

Comment:
This class is causing a horizontal scrolling bar to appear (and it is not compensated in regards to the main content)